modules/core/src/main/scala/gem/enum/package.scala

```diff
@@ -30,4 +34,38 @@ package object enum {
         case SmartGcalType.NightBaseline => baseline(GcalBaselineType.Night)
       }
   }
+
+  @SuppressWarnings(Array("org.wartremover.warts.Throw"))
+  implicit class HalfCompanionOps(companion: Half.type) {
+
+    def unsafeFromInt(n: Int): Half =
```

Comment:
This made me realize that I don't see the definition of `Half`. Why do you need a `toInt` method? for serializing to the db?